src/postprocessing/Pass.ts

```diff
@@ -0,0 +1,52 @@
+import { OrthographicCamera, PlaneGeometry, Mesh, Material, Renderer } from 'three'
+
+function Pass() {
+  // if set to true, the pass is processed by the composer
+  this.enabled = true
+
+  // if set to true, the pass indicates to swap read and write buffer after rendering
+  this.needsSwap = true
+
+  // if set to true, the pass clears its buffer before rendering
+  this.clear = false
+
+  // if set to true, the result of the pass is rendered to screen. This is set automatically by EffectComposer.
+  this.renderToScreen = false
+}
+
+Object.assign(Pass.prototype, {
+  setSize: function (/* width, height */) {},
+
+  render: function (/* renderer, writeBuffer, readBuffer, deltaTime, maskActive */) {
+    console.error('THREE.Pass: .render() must be implemented in derived pass.')
+  },
+})
+
+// Helper for passes that need to fill the viewport with a single quad.
+class FullScreenQuad {
+  camera = new OrthographicCamera(-1, 1, 1, -1, 0, 1)
+  geometry = new PlaneGeometry(2, 2)
+  _mesh: Mesh
+
+  constructor(material: Material) {
```

Comment:
IMO this should be a generic `TMaterial extends Material = Material` (which will go in the class declaration and used here) that way we know what material it is when people use it because some materials have properties that others don't and Typecasting is nice to be avoided when possible

Reply:
good call - done